### PR TITLE
change scripts to Menu for all menu's

### PR DESCRIPTION
**Pull request by szaimen**
as the title says

### CODE REVIEW

File: menu/additional_apps.sh [hunk 1]
```diff
@@ -114,7 +114,7 @@ case "$choice" in
     ;;&
     *"Talk"*)
         clear
-        print_text_in_color "$ICyan" "Downloading the Talk script..."
+        print_text_in_color "$ICyan" "Downloading the Talk MENU..."
```
Comment (enoch85) [inline menu/additional_apps.sh:117]: MENU Menu

File: menu/additional_apps.sh [hunk 2]
```diff
@@ -45,17 +45,17 @@ $CHECKLIST_GUIDE\n\n$RUN_LATER_GUIDE" "$WT_HEIGHT" "$WT_WIDTH" 4 \
 case "$choice" in
     *"Documentserver"*)
         clear
-        print_text_in_color "$ICyan" "Downloading the Documentserver script..."
+        print_text_in_color "$ICyan" "Downloading the Documentserver MENU..."
```
Comment (enoch85) [inline menu/additional_apps.sh:48]: MENU Menu

File: menu/additional_apps.sh (hunk 1 above)
Comment (szaimen) [reply menu/additional_apps.sh]: Done

File: menu/additional_apps.sh (hunk 2 above)
Comment (szaimen) [reply menu/additional_apps.sh]: Done

**Comment by enoch85** [comment]
Thanks! :+1:

**Pull request merged**